_data/day1-menus.yml

```diff
@@ -0,0 +1,25 @@
+Breakfast:
+    Options:
+        "Cranberry Orange Yogurt Parfait with Cinnamon-Spiced Pumpkin Seeds and Dried Cherries": "GF"
+        "Almond Joy chia chocolate pudding topped with toasted coconut and roasted almonds": "VGN, GF"
+        "Mini NY-Style Bagels with Spreads": ""
+        "Freshly Baked Muffins": ""
+        "Sliced Fresh Fruit": ""
+        "Assorted Juices": ""
+        "Additional GF Breakfast Items": "GF"
+    Beverages:
+        "Princeton Premium Blend Regular and Decaffeinated Coffee": ""
+        "Assorted teas": ""
+Lunch:
+    Entrees:
+        "Roasted Spiced Delicata Squash, Baby Tuscan Kale, Toasted Pepita Seeds, Goat Cheese, and Mustard Seed Vinaigrette": "VGT"
+        "Shaved Brussels Sprouts Salad, Red Cabbage, Pickled Carrots, Sunflower Seeds, Dried Cranberries, and Sweet Creamy Celery Dressing": "VGN"
```

Comment:
I think this item got dropped here: Grilled Cremini Mushrooms, Roasted Fennel, Flat Leaf Parsley, Crumbled Toasted Pine Nuts, Sherry Wine Vinaigrette (Vegan)

Reply:
Thanks! Fixed in e33ed5c